nntrainer/layers/layer_node.h

```diff
@@ -109,6 +119,7 @@ class LayerNode : public ml::train::Layer, public GraphNode {
 #endif
 
 private:
+  // TODO: make this unique_ptr once getObject API is removed
```

Comment:
fyi) putting this to unique_ptr will prohibit copying this class but moving is okay as we implement it.

Reply:
yes.